server/Middleware/auth.js

```diff
@@ -0,0 +1,35 @@
+import jwt from 'jsonwebtoken';
+import db from '../database/queries'
+import executor from '../database/queryExecutor';
+
+const Auth = {
+    async verifyToken(req, res, next) {
+        const token = req.headers['access-token'];
+        if (!token) {
+            return res.status(400).send({
```

Comment:
The status code here should be `401` since users without `Tokens` are unauthorized.

Reply:
Status code changed